src/main/java/domain/card/Deck.java

```diff
@@ -0,0 +1,42 @@
+package domain.card;
+
+import domain.user.Dealer;
+import domain.user.Players;
+import domain.user.User;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class Deck {
+    private static final int ZERO = 0;
+    private static final int NUMBER_OF_CARDS_BY_RULE = 2;
+    private static final int TOP = 0;
+
+    private static List<Card> cards = new ArrayList<>();
+
+    public static void initializeDeck(CardShuffler cardShuffler) {
+        cards.clear();
+        for (Symbol symbol : Symbol.values()) {
+            Arrays.stream(Type.values()).forEach(type -> cards.add(new Card(symbol, type)));
```

Comment:
가급적  가독성을 위해서 아래와 같이 ``stream()``  다음에는 들여쓰기를 해주세요 🙏
이와 유사한 부분들은 찾아서 전부 수정해주세요 😉

```suggestion
            Arrays.stream(Type.values())
                .forEach(type -> cards.add(new Card(symbol, type)));
```

Reply:
반영하도록 하겠습니다 :)